Tests/SwiftyRequestTests/SwiftyRequestTests.swift

```diff
@@ -1,5 +1,5 @@
 import XCTest
-import CircuitBreaker
+import CircuitBreakerSwift
```

Comment:
This didn't make sense to me... We can't change the modules vended by packages, that would be an API-breaking change. Can you explain?

Reply:
Sorry, was just experimenting with different methods of creating a viable Pod for the repo, but managed to resolve it without going this route. I shall delete this PR and branch as they aren't needed anymore.